components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.policy.create/public/js/policy-create.js

```diff
@@ -428,8 +428,10 @@ var savePolicy = function (policy, isActive, serviceURL) {
             $(".add-policy").addClass("hidden");
             $(".policy-naming").addClass("hidden");
             $(".policy-message").removeClass("hidden");
-            setTimeout(function(){}, 1000);
-            window.location.href = "/devicemgt/policies";
+            setTimeout(function(){
```

Comment:
@menakaj There is a formatting issue here can we fix this? I have noticed few more formatting issues in the PR as well. Please go through and fix them.